esp/src/eclwatch/LockDialogWidget.js

```diff
@@ -97,6 +97,7 @@ define([
 
                 if (this.unlockForm.validate()) {
                     cookie("Status", "login_attempt");
+                    cookie("ESPSessionReq", "true");
```

Comment:
This cookie is not needed for log out because ESP will add the cookie before the login.html is displayed. As soon as the logout, ESP receives a 'GET /' request. ESP will add the cookie before the request is forwarded to the login.html.